<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Derivatives</class>
 <widget class="QDialog" name="Derivatives">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>966</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Derivatives Lab</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>1141</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Random Walk</string>
    </attribute>
    <widget class="MPL_WIDGET" name="oMplCanvas" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>851</width>
       <height>671</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>360</y>
       <width>0</width>
       <height>321</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="MPL_WIDGET" name="oMplCanvas2" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>10</y>
       <width>261</width>
       <height>451</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNum_prob">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>750</y>
       <width>40</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Copy2Clipboard">
     <property name="geometry">
      <rect>
       <x>994</x>
       <y>490</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Copy to Clipboard</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei UI</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#aa0000;&quot;&gt;NTUST Derivatives Lab&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#aa0000; vertical-align:super;&quot;&gt;TM &lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#5555ff;&quot;/&gt;&lt;span style=&quot; font-size:14pt; color:#313194;&quot;&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;Option Pricing and Strategies 選擇權定價與策略 Module 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>920</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QSlider" name="slider_t2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>930</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_t1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>900</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>900</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>930</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>890</y>
     <width>114</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>label_t1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>920</y>
     <width>114</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>label_t2</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>951</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="title">
    <string>Parameters</string>
   </property>
   <widget class="QSlider" name="oSlider_prob">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_p">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>22</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>p = </string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>20</y>
      <width>50</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_SimSize">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>55</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Simulation size</string>
    </property>
    <item>
     <property name="text">
      <string>100000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>26</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QSlider" name="oSlider_u">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>105</number>
    </property>
    <property name="sliderPosition">
     <number>105</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_u">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>13</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Δx =&lt;br/&gt;u =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="oSlider_PathsShown">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="sliderPosition">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSlider" name="oSlider_TimeStep">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>No. of time steps</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="sliderPosition">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_P">
    <property name="geometry">
     <rect>
      <x>623</x>
      <y>24</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>No. of simulated paths to show</string>
    </property>
    <property name="text">
     <string>Paths = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_T">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>26</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>n  =</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_S0">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_S0">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>16</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X0 = ln(Y0) = 
Y0 =</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_K">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_K">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>15</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ln(K) =&lt;br/&gt;K = &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioRW">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>RW</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioGRW">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>71</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>GRW</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_ShowRandomWalk">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Simulate </string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPL_WIDGET</class>
   <extends>QWidget</extends>
   <header location="global">mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Derivatives</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Derivatives</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
